### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
-        "silverstripe/cms": "^4.4",
-        "silverstripe/admin": "^1.4",
-        "silverstripe/vendor-plugin": "^1",
-        "symbiote/silverstripe-gridfieldextensions": "^3.2",
+        "php": "^8.1",
+        "guzzlehttp/guzzle": "^7.5",
+        "silverstripe/cms": "^5",
+        "silverstripe/admin": "^2",
+        "silverstripe/vendor-plugin": "^2",
+        "symbiote/silverstripe-gridfieldextensions": "^3.5",
```
Comment (emteknetnz) [inline composer.json]:
```suggestion
        "symbiote/silverstripe-gridfieldextensions": "^4",
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (sabina-talipova) [reply composer.json]: DONE

**Review by emteknetnz — DISMISSED** [review]
Failing unit tests

**Comment by sabina-talipova** [comment]
Created new issue for failed Behat tests https://github.com/silverstripe/silverstripe-ckan-registry/issues/280

File: tests/php/Page/CKANRegistryPageControllerFunctionalTest.php [hunk 2]
```diff
@@ -24,15 +22,17 @@ protected function setUp(): void
 
     public function testGetSchemaFromPageUrl()
     {
-        $response = $this->get('animal-centers/schema');
+        $this->logInWithPermission('ADMIN');
```
Comment (maxime-rainville) [inline tests/php/Page/CKANRegistryPageControllerFunctionalTest.php]:
```suggestion
        $this->logInWithPermission('VIEW_DRAFT_CONTENT');
```

File: tests/php/Page/CKANRegistryPageControllerFunctionalTest.php [hunk 3]
```diff
@@ -24,15 +22,17 @@ protected function setUp(): void
...
+        $response = $this->get('animal-centers/schema?stage=Stage');
 
         $this->assertSame(200, $response->getStatusCode());
         $this->assertJson($response->getBody());
     }
 
     public function testGetSchemaFromPageSubUrl()
     {
-        $response = $this->get('animal-centers/view/123/schema');
+        $this->logInWithPermission('ADMIN');
```
Comment (maxime-rainville) [inline tests/php/Page/CKANRegistryPageControllerFunctionalTest.php]:
```suggestion
        $this->logInWithPermission('VIEW_DRAFT_CONTENT');
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Just need to fix the permission check.

File: tests/php/Page/CKANRegistryPageControllerFunctionalTest.php (hunk 2 above)
Comment (sabina-talipova) [reply tests/php/Page/CKANRegistryPageControllerFunctionalTest.php]: DONE

File: tests/php/Page/CKANRegistryPageControllerFunctionalTest.php (hunk 3 above)
Comment (sabina-talipova) [reply tests/php/Page/CKANRegistryPageControllerFunctionalTest.php]: DONE

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**